server/app/assets/javascripts/preview.ts

```diff
@@ -84,6 +86,19 @@ class PreviewController {
       )
       PreviewController.updateFromNewQuestionHelpText(helpTextInput.value)
     }
+    const defaultState = document.getElementById(
+      PreviewController.QUESTION_ADDRESS_DEFAULT_STATE,
+    ) as HTMLInputElement | null
+    if (defaultState) {
+      defaultState.addEventListener(
+        'input',
+        (ev) => {
+          PreviewController.updateDefaultStateText(defaultState.value)
+        },
+        false,
```

Comment:
Don't need to pass `false` here. This field ([useCapture](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener)) defaults to false anyway.